pkgs/top-level/python-packages.nix

```diff
@@ -17597,6 +17597,8 @@ EOF
 
   simpy = callPackage ../development/python-modules/simpy { };
 
+  fire = callPackage ../development/python-modules/fire { };
```

Comment:
This file should be ordered alphabetically.

Reply:
That file is actually so chaotic and long that one would be inclined to find another ordering mechanism. Search is the only reasonable way of finding any package...

I've swapped it somewhere where it seems more alphabetic...